manifests/params.pp

```diff
@@ -441,7 +441,7 @@
   $server_idle_timeout                    = 1200000
   $server_web_idle_timeout                = 30000
   $server_connect_timeout                 = 120000
-  $server_enable_ruby_profiler            = false
+  $server_enable_ruby_profiler            = true
```

Comment:
the default changed for the metrics (which are also on by default) - maybe that paramter should go and this get enabled and disabled with metrics?